README.md

```diff
@@ -16,17 +19,16 @@ So, follow common conventions with respect to directory structure and names to m
 ### Candidates where programming is required (Data Scientist and Engineers)
 Requirements and notes:
 * Our primary programming languages are `python` and `R`. We will accept code that is packaged in `py`, `.ipynb`, `.R` and `.Rmd` files. 
-* Bash scripts are fine for glue. You may develop in any development environment you choose. 
+* Bash or similar scripting language files are fine for glue. You may develop in any development environment you choose. 
 * We expect to be able to clone your repo, immediately identify what script to execute from your README file, and execute it to completion with no human interaction. 
   In order to ensure that our environment has the right libraries or packages, please follow standard python (PEP8) or R guidelines for structure in your code, i.e place `import` and `library()` commands at the top of your scripts.
 * If your repo does not clone and run, we will not attempt to fix it.
 * If your analysis makes use of any external data, the data must either be included in the repo, or be downloaded automatically during script execution.
 
 ### Candidates where programming is not required (Data Analysts)
-*NB* If you prefer, you may submit using the requirements described above.
+*Note* If you prefer, you may submit using the requirements described above.
 
-The final output of your analysis should be either a self-contained `html` file, executed `ipynb` file, 
-Excel document with appropriate formatting and pivot tables, or a PowerBI file.
+You can use any tool to produce the output, e.g. Excel, Power BI, Tableau, etc. The final deliverable needs to be a pdf report with your analysis.
```

Comment:
can we include python and R as examples?  You can use any tool to produce the output, e.g. Excel, Power BI, Tableau, Python, R etc. ......................